internal/platform/database/repair/repository_test.go

```
@@ -0,0 +1,185 @@
+			"Bathroom leak",
+			"cancelled",
+			now,
+			nil,
+			nil,
+			cancelReason,
+			now,
+			now,
+		))
+	mock.ExpectCommit()
+
+	tx, err := db.Begin()
```

golangci-lint (2.11.4)

[error] (*database/sql.DB).Begin must not be called. use (*database/sql.DB).BeginTx

(noctx)